lib/sensors/tof.cpp

```diff
@@ -48,40 +48,32 @@ core::Result TimeOfFlight::initialise()
     // i2c_->writeByteToRegister(device_address_, 0x0207, 0x01);
     // i2c_->writeByteToRegister(device_address_, 0x0208, 0x01);
 
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[0], 0x00);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[1], 0xfd);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[2], 0x01);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[3], 0x03);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[4], 0x02);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[5], 0x01);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[6], 0x03);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[7], 0x02);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[8], 0x05);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[9], 0xce);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[10], 0x03);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[11], 0xf8);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[12], 0x00);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[13], 0x3c);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[14], 0x00);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[15], 0x3c);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[16], 0x09);
-    i2c_->writeByteToRegister(device_address_, array_return_addresses[17], 0x09);
-    // i2c_->writeByteToRegister(device_address_, 0x0198, 0x01);
```

Comment:
These registers were not commented out because they should be removed. They were commented out because at the time this code was written, #58 hadn't been merged yet. As such, the 16-bit registers being written to would cause build failure.
These registers should be added back in. Refer to the application note in the PR's header